.nvmrc

```
@@ -0,0 +1 @@
+16.10.0
```

```suggestion
16
```

just go with the major, as all the LTS line has compatibility (we're probably not using newer features in the 16.x line)